docs/introduction.md

```diff
@@ -1,42 +1,43 @@
 # Introduction
 
-deck.gl was created to make "reactive" WebGL-based visualizations of
-"big data" simple, and to enable new users to quickly get impressive
-results, while gradually enabling more flexibility and power as
-users climb the learning curve.
+Deck.gl is a WebGL-powered framework that features an instancing and layering
+paradigm. It is specifically designed for composing and visualizing data sets
+at scale. With its reactive API design, deck.gl makes visual analytics of Big
+Data simple. It enables users to quickly prototype and get impresssive results,
+while gradually unlocking more flexibility and power as users climb the
+learning curve.
 
 
 ## Brief Overview
 
-The basic idea is to render a stack of visual overlays, usually (but
-not always) over maps.
+The basic idea of using deck.gl is to render a stack of [visual] overlays,
+usually (sbut not necessarily) over maps.
```

Comment:
was trying to convey that it is optional that the layer is visible, layers like voronoi could be functional layers. it seems the square bracket is not self-explanatory, will remove it and keep it as visual overlays for now.